NetflixClone/Classes/Models/Movie.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Movie.swift
+//  NetflixClone
+//
+//  Created by Joseph Ching on 2022-09-27.
+//
+
+import Foundation
+
+struct Movie: Codable, Hashable {
```

Comment:
```suggestion
struct Movie: Decodable, Hashable {
```
Since we will use it for decoding purposes only.